rules/coding-style/tests/Rector/String_/UseClassKeywordForClassNameResolutionRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Rector\CodingStyle\Tests\Rector\String_\UseClassKeywordForClassNameResolutionRector\Fixture;
+
+class SomeClassÿ
+{
+    public static function createCache()
+    {
+        return 'cache';
+    }
+}
+
+class AnotherClass
+{
+    private function write($first)
+    {
+
+    }
+
+    public function get()
+    {
+        $this->write(
+            '<?php if (Rector\CodingStyle\Tests\Rector\String_\UseClassKeywordForClassNameResolutionRector\Fixture\SomeClassÿ::createCache($netteCacheStorage, %var,  $this->global->cacheStack, %node.array?)) { ?>'
+        );
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\CodingStyle\Tests\Rector\String_\UseClassKeywordForClassNameResolutionRector\Fixture;
+
+class SomeClassÿ
```

Comment:
Testing for weird chars it not really needed. We have to only cover saint cases :)

Reply:
It was rather for me to make sure regex is okay :)
I'll rename it.

Reply:
Renamed